website/client/src/components/userMenu/profile.vue

```diff
@@ -766,7 +771,7 @@ export default {
       achievements: {},
       achievementsCategories: {}, // number, open
       content: Content,
-      user: undefined,
+      user: {},
```

Comment:
why not keep undefined? This way `v-if="!user"` will never evaluate to true because `{}` is considered a truthy value

Reply:
This was to keep 404 from flashing on modal. I set user to undefined only if 'fetchMember' returns a 404.

If it wasn't for flashing 404 bug I wouldn't have done this.

Reply:
Would you like me to try and find another way to fix the flash 404 bug without changing the initial state of user?

Reply:
sorry for the delay, yes if you could try another way it would be great but if it's not possible let me know

Reply:
Im having a hard time finding a way around this. Any suggestions?